jwst/pathloss/pathloss.py

```diff
@@ -10,9 +10,10 @@
 log.setLevel(logging.DEBUG)
 
 
-# There are 30 slices in the NIRSPEC IFU, numbered from 0 to 29
+# There are 30 slices in the NIRSpec IFU, numbered from 0 to 29
```

Comment:
A LOT of mods in this module are purely cosmetic, just for readability, flake8, etc. Only substantive changes directly involve `pathloss_ps` and `pathloss_un`.